sympy/solvers/ode/tests/test_single.py

```diff
@@ -1785,25 +1785,25 @@ def _get_examples_ode_sol_separable():
         'sol': [Eq(f(x), (-C1 - x**2)/(-C1 + x**2))],
     },
 
-    # https://github.com/sympy/sympy/issues/10379
-    'separable_24': {
-        'eq': f(t).diff(t)-(1-51.05*y*f(t)),
-        'sol': [Eq(f(t), (0.019588638589618023*exp(y*(C1 - 51.049999999999997*t)) + 0.019588638589618023)/y)],
-        'func': f(t),
-    },
+    # # https://github.com/sympy/sympy/issues/10379
+    # 'separable_24': {
+    #     'eq': f(t).diff(t)-(1-51.05*y*f(t)),
+    #     'sol': [Eq(f(t), (Float("2.783712970656612e-16")*exp(y*(C1 - 51.05*t)) + Float("0.01958863858961802"))/y)],
+    #     'func': f(t),
+    # },
```

Comment:
There was a significant amount of error in this calculation originally. The new result approximates the rational solution correctly.